src/test/java/com/unit/UNIT/TEST/MachineTest.java

```diff
@@ -109,4 +110,54 @@ public void shouldNotIncreaseAmountOfMoney() {
         assertEquals(finalResult, machine.bankAmount);
         verify(validator,times(1)).validateMoney(money);
     }
+
+    @Test
+    public void shouldPlayerLose() {
+        //given
+        machine.gameSigns = IntStream.of(1, 2, 3, 4, 5).boxed().collect(Collectors.toList());
+        //when
+        machine.checkWin();
+        //then
+        assertFalse(machine.checkWin());
+    }
+
+    @Test
+    public void shouldAssignFiveToBankAmount() {
+        //given
+        Money money = new Money(5);
+        when(validator.validateMoney(money)).thenReturn(true);
+        int expectedAmount = 5;
+        //when
+        machine.winInfo(); // Zeroes bank amount
```

Comment:
Tego już nie poprawiaj, szkoda Twojego czasu, a jedynie zapamiętaj: Komentarz umieszczasz tylko wtedy jak jest on niezbędny! (np. masz pustą klasę i piszesz //this method must be empty, because it initialize component albo //this method must be empty due to fixing Sonar Issues (narzędzie do sprawdzania czystości kodu).